src/main/java/seedu/address/ui/DeliverableDetailsPanel.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.deliverable.deliverable.Deliverable;
+
+/**
+ * Panel containing the expanded details of a deliverable.
+ */
+public class DeliverableDetailsPanel extends UiPart<Region> {
+    private static final String FXML = "DeliverableDetailsPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(DeliverableDetailsPanel.class);
+
+    @FXML
+    private Label title;
+    @FXML
+    private Label milestone;
+    @FXML
+    private Label description;
+    @FXML
+    private Label deadline;
+    @FXML
+    private VBox contactList;
+    @FXML
+    private Label isCompleted;
+
+    /**
+     * Creates a {@code DeliverableDetailsPanel} with the given {@code Deliverable}
+     */
+    public DeliverableDetailsPanel(Deliverable deliverable) {
+        super(FXML);
+        assert deliverable != null;
+        title.setText(deliverable.getTitle().value);
+        // title.setWrapText(true);
+        milestone.setText((deliverable.getMilestone().toString()));
+        description.setText(deliverable.getDescription().toString());
+        // description.setWrapText(true);
+        deadline.setText(deliverable.getDeadline().value);
+        deliverable.getContacts().value.ifPresentOrElse(contacts -> {
```

Comment:
might want to abstract this into a method that MeetingDetailsPanel can also use

Reply:
But doesnt meeting have different attributes